gr-blocks/include/gnuradio/blocks/keep_one_in_n.h

```diff
@@ -30,7 +30,7 @@ namespace gr {
   namespace blocks {
 
     /*!
-     * \brief decimate a stream, keeping one item out of every \p n.
+     * \brief decimate a stream, keeping the last item out of every \p n.
```

Comment:
Wow, is this what it does? Might want to consider picking the first, for consistency with keep M in N and probably most people's intuition.